components/security/authentication.rst

```diff
@@ -298,6 +299,18 @@ a ``security.authentication.failure`` event is dispatched. You could listen on
 the ``security.authentication.failure`` event, for example, in order to log
 failed login attempts.
 
+Authentication Sensitive Event
+------------------------------
+
```

Comment:
We should add a directive saying this is new in Symfony 4.3:

```rst
.. versionadded:: 4.3

    The ``security.authentication.success_sensitive`` was introduced in Symfony 4.3.

Reply:
Done!